### ISSUE TITLE
Float number support for data source

### ISSUE THREAD

**Issue by amoshsueh** [Boris-Em/BEMSimpleLineGraph#64]
Hello

Does the data source support float number? 
I tried to add some float numbers but the Y-axis label is out of the chart view and the popup report label only can show integer number.
```
NSArray *numberArray = @[@35.2f, @35.0f, @35.3f, @35.5f, @35.4f, @35.2f, @35.4f, @36.0f, @35.9f, @35.5f, @35.1, @34.9f, @35.0f];
```
![rsz_ios_simulator_screen_shot_201493_ 11111](https://cloud.githubusercontent.com/assets/8340691/4122513/52863adc-32c5-11e4-86fb-a676f208da78.png)

**Comment by inidy** [comment]
Does X- axis label is cutting off for anyone?

**Comment by Sam-Spencer** [comment]
Closed with merge of Pull Request #132.
